package.json

```diff
@@ -1,37 +1,21 @@
 {
-  "name": "clerk-fastify-starter",
-  "description": "Official starter repo for Clerk and Fastify",
+  "name": "clerk-fastify-quickstart",
   "private": true,
-  "keywords": [
-    "clerk",
-    "nextjs",
-    "fastify",
-    "authentication"
-  ],
+  "type": "module",
   "scripts": {
-    "start": "npx ts-node src/index.ts",
+    "start": "NODE_OPTIONS=--loader=ts-node/esm npx ts-node src/index.ts",
     "start:runtime-keys": "npx ts-node src/using-runtime-keys.ts",
     "start:specific-routes": "npx ts-node src/authenticating-specific-routes.ts",
-    "dev": "npx nodemon src/index.ts"
+    "dev": "NODE_OPTIONS=--loader=ts-node/esm nodemon --exec npx ts-node src/index.ts"
   },
   "dependencies": {
-    "@clerk/fastify": "latest",
-    "dotenv": "^16.0.3",
-    "fastify": "4.12.0"
+    "@clerk/fastify": "^1.0.45",
+    "dotenv": "^16.4.5",
+    "fastify": "^4.28.1"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/clerkinc/clerk-fastify-starter.git"
-  },
-  "author": "Clerk Inc",
-  "bugs": {
-    "url": "https://github.com/clerkinc/clerk-fastify-starter/issues"
-  },
-  "homepage": "https://github.com/clerkinc/clerk-fastify-starter#readme",
   "devDependencies": {
-    "@types/dotenv": "^8.2.0",
-    "nodemon": "^2.0.20",
-    "prettier": "^2.8.4",
-    "ts-node": "^10.9.1"
+    "nodemon": "^3.1.6",
+    "ts-node": "^10.9.1",
```

Comment:
Considering if we should continue using `ts-node` with `nodemon`, or just use something like [`tsx`](https://github.com/privatenumber/tsx) and take advantage of the `watch` option:

```bash
tsx watch src/index.ts
```

I thought of this because we'll be adding an Express quickstart later as well.